modules/apm/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -89,3 +89,5 @@ elastic.apm.agent:
       properties:
         - AsyncProfiler.safemode
   - load_native_libraries
+  - manage_threads
+  - outbound_network
```

Comment:
wondering, why is elastic.apm.agent needed at all? shouldn't this be covered by the default agent policy?
a bit worrying is that we receive metrics from my QA projects despite not allowing `outbound_network` here or in the default agent policy 😱

Reply:
Were they apm metrics or just generic container metrics provided by metricbeat on the host?

Reply:
The issue about missing outbound_network but still working might be due to missing instrumentation to URLConnection & derived that we discovered.
> why is elastic.apm.agent needed at all

Hypothesis (that I will try to confirm): in the plugin that jar/module is used kind of a client/an API to talk to the agent/to the server?

Reply:
But I do wonder if we need these policies in both places, or just here, or just on the agentEntitlements fixed set.
I'll dig a bit more on this point too.

Reply:
I moved these entitlements to the `agentEntitlements` policy; considering the dependency is runtimeOnly and the code using these entitlements is in the agent, seems like that is the right place for them.
Tests pass, even though APM tests are _really_ limited - I tried to run ES with-apm-server too and no errors there.
Will also try a Serverless QA deployment before merging.